### PR TITLE
Replace cascade=false -> cascade=orphan

### LINKED ISSUE #28890: DaemonSet: kubectl delete --cascade=false
According to https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#delete the `kubectl delete` can get a parameter `--cascade` with a value of "background", "orphan", or "foreground".

https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#updating-a-daemonset proposes to use `--cascade=false`.

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #28890

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/use-cascading-deletion.md [hunk 1]
```diff
@@ -302,7 +302,7 @@ For details, read the [documentation for your Kubernetes version](/docs/home/sup
 Run the following command:
 
 ```shell
-kubectl delete deployment nginx-deployment --cascade=false
+kubectl delete deployment nginx-deployment --cascade=orphan
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/use-cascading-deletion.md:305]: Hi @Patil2099 .
Just checking if this change is correct for Kubernetes versions *prior* to 1.20.x?
When did the kubectl flag change?

**Comment by celestehorgan** [comment]
@kubernetes/sig-cli-pr-reviews please review this for accuracy

File: content/en/docs/tasks/administer-cluster/use-cascading-deletion.md (hunk 1 above)
Comment (eddiezane) [reply content/en/docs/tasks/administer-cluster/use-cascading-deletion.md:305]: Prior to 1.20 this was a boolean flag.

https://github.com/kubernetes/kubernetes/pull/93384/files#diff-3b24d71dec02e50031772f99bffc240106e8098d0a261efb10cd838b6a14a467L122

**Review by soltysh — APPROVED** [review]
> @kubernetes/sig-cli-pr-reviews please review this for accuracy

lgtm from cli pov

File: content/en/docs/tasks/administer-cluster/use-cascading-deletion.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/tasks/administer-cluster/use-cascading-deletion.md:305]: OK. Thanks @eddiezane .
The page preview:
https://deploy-preview-29394--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/use-cascading-deletion/#set-orphan-deletion-policy

I guess the change in this page should be reverted?
@Patil2099 , Does this make sense?

**Comment by reylejano** [comment]
I confirmed `--cascade` (tested on 1.20, 1.21, 1.22) requires either `background`, `orphan`, or `foreground` to set the deletion cascading strategy for dependencies

There are lgtms from @tengqm form SIG Docs and @soltysh from SIG CLI

/approve

**Comment by kbhawkey** [comment]
Thanks @reylejano !

**Pull request merged**